pom.xml

```diff
@@ -270,7 +270,7 @@
       <plugin>
         <groupId>com.spotify.fmt</groupId>
         <artifactId>fmt-maven-plugin</artifactId>
-        <version>2.20</version>
+        <version>2.25</version>
```

Comment:
Unrelated, but good to bump. It also fixes some dependency conflicts when running Java 21 (on my local machine, at least)